examples/server/webui/src/components/ChatScreen.tsx

```diff
@@ -259,8 +267,7 @@ export default function ChatScreen() {
             className="textarea textarea-bordered w-full"
             placeholder="Type a message (Shift+Enter to add a new line)"
             ref={inputRef}
-            value={inputMsg}
-            onChange={(e) => setInputMsg(e.target.value)}
+            defaultValue={prefilledMsg.content()}
```

Comment:
There could be a race condition between `prefilledMsg.content()` and `prefilledMsg.clear()` above